app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6

```diff
@@ -213,11 +233,7 @@ Object.assign(MediaElementPlayer.prototype, {
     },
 
     isSamePlaylistItem(el) {
-      return (
-        $(el)
-          .parent('li')
-          .data('playlistItemId') === this.$nowPlayingLi.data('playlistItemId')
-      );
+      return $(el).data('playlistItemId') === this.$nowPlayingLi.data('playlistItemId');
```

Comment:
You use `dataset.playlistItemId` elsewhere in this PR.  Can you use it here?

Reply:
javascript vs jquery. different types of objects require different methods of access